authorization/ngsild-payloads/Group.json

```diff
@@ -0,0 +1,34 @@
+{
+   "id":"urn:ngsi-ld:Group:5acb5d63-fd74-4ad0-af5e-dc1fc1d4bc29",
+   "type":"Group",
```

Comment:
What about the expansion/reduction of the Goup/User types ?
With the default context ?

Reply:
done